docs/strategy-customization.md

```diff
@@ -639,6 +639,157 @@ Stoploss values returned from `custom_stoploss` must specify a percentage relati
 
     Full examples can be found in the [Custom stoploss](strategy-advanced.md#custom-stoploss) section of the Documentation.
 
+!!! Note
+    Providing invalid input to `stoploss_from_open()` may produce "CustomStoploss function did not return valid stoploss" warnings.
+    This may happen if `current_profit` parameter is below specified `open_relative_stop`. Such situations may arise when closing trade
+    is blocked by `confirm_trade_exit()` method. Warnings can be solved by never blocking stop loss sells by checking `sell_reason` in
+    `confirm_trade_exit()`, or by using `return stoploss_from_open(...) or 1` idiom, which will request to not change stop loss when
+    `current_profit < open_relative_stop`.
+
+### *stoploss_from_absolute()*
```

Comment:
I'm not sure in what usecase you'd want to define an absolute stoploss - it'll always need to depend on the entry price (you can't generally say - stoploss when BTC hits 48k - it's simply not going to work with absolute values as prices will change over time).

You might wanna define it based on profit ... like "we're now +10% - set stoploss at +5% (so worst case, that trade is 5% winner) - but let it run for now - where we have `stoploss_from_open()`.

Even in the sample below, what you really want to say is "at 7% profit" - not **at 107$**.
Using absolute values will be difficult/impossible to generalize this if you have coins that cost 107$ - and others that cost 40.000$ - 7$ for one is 7% - while for the other it's not even 1% ...

Reply:
```py
dataframe['stoploss'] = dataframe['close'] - dataframe['atr'] * 2
...
return stoploss_from_absolute(last_candle['stoploss'], current_rate)
```
And usecase is not only involving ATR, but anything that involves absolute price (stop at moving average, lower channel/band).

Reply:
Then this should be the example, not one that is using relative pricing to get an absolute price from open - which is where `stoploss_from_open()` will be easier to use, and makes this appear non-necessary.

we'll also want a test for this ... (i'm not sure why it's in this PR - it feels "snuck" in somehow :laughing:  - it's not related to informative decorator apart from having documentation nearby...).

Reply:
> i'm not sure why it's in this PR - it feels "snuck" in somehow laughing - it's not related to informative decorator apart from having documentation nearby...

True. `@informative` was supposed to be a simple helper too, but it somehow snowballed.... 😊